tracer/src/Datadog.Trace/Ci/TestSession.cs

```diff
@@ -426,7 +426,7 @@ internal bool InternalClose(TestStatus status, TimeSpan? duration)
         {
             foreach (var eVar in envVars)
             {
-                EnvironmentHelpers.SetEnvironmentVariable(eVar.Key, eVar.Value);
+                EnvironmentHelpers.SetEnvironmentVariable(eVar.Key, eVar.Value ?? string.Empty);
```

Comment:
I'm not sure this one is correct, Environment.SetEnvironmentVariable accepts a null value: https://learn.microsoft.com/en-us/dotnet/api/system.environment.setenvironmentvariable?view=net-8.0#system-environment-setenvironmentvariable(system-string-system-string)

Reply:
Yeah, that looks unnecessary, _especially_ as this is calling `EnvironmentHelpers`, which already accepts a null 😅